config/rewrites.json

```diff
@@ -63,6 +63,21 @@
             "status": 301
         }
     ],
+    "getcarina.com": [
+        {
+            "from": "^\\/blog\\/?$",
+            "to": "^\\blog\\/announcing-carina\\/",
+            "rewrite": false,
+            "status": 301
```

Comment:
@ktbartholomew Actually, should I make this a 302, so the redirect doesn't get cached once we _do_ have a blog landing page?